bsdlib/README.rst

```diff
@@ -87,6 +87,48 @@ The figure below shows a simplified BSD library architecture.
 
    BSD library architecture diagram
 
+
+Security tag model
+******************
+
+The modem on an nRF91 device is equipped with a full IPv4/IPv6 stack and a DTLS/TLS engine.
+To use the cryptographic functions in the modem, an application must provision the security assets to the modem before using them.
+This can be done offline or while the modem is in offline mode.
+The security assets are provisioned through AT commands and referenced later by a security tag (sec_tag) when a DTLS/TLS socket is created.
```

Comment:
Could link to LTE Link Monitor here? https://infocenter.nordicsemi.com/topic/ug_link_monitor/UG/link_monitor/lm_certificate_manager.html